src/containers/BooksList.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { createBook, removeBook } from '../actions/index';
+import Book from '../components/Book';
+
+class BooksList extends React.Component {
+  render() {
+    const Books = this.props.books.map((book, idx) => 
+      <Book key={idx} id={book.id} title={book.title} category={book.category} />
```

Comment:
It is not a good idea to use index as key in react lists as it needs to identify elements uniquely. See [this article](https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318).
`book.id` would be great as key here.